posthog/models/app_metrics/sql.py

```diff
@@ -64,9 +65,24 @@
     error_details String CODEC(ZSTD(3))
 )
 ENGINE={kafka_engine(topic=KAFKA_APP_METRICS)}
+SETTINGS kafka_handle_error_mode='stream'
 """
 )
 
+# MergeTreeEngine(table_name, replication_scheme=ReplicationScheme.REPLICATED)
+KAFKA_APP_METRICS_DLQ_SQL = lambda: KAFKA_ENGINE_DLQ_BASE_SQL.format(
+    table="kafka_app_metrics_dlq",
+    cluster=settings.CLICKHOUSE_CLUSTER,
+    engine=MergeTreeEngine("kafka_app_metrics_dlq", replication_scheme=ReplicationScheme.REPLICATED),
```

Comment:
Q: Why replicated, not sharded?

Reply:
Happy to change this, but honestly for complexity. I don't think these tables will have too much data to warrant a sharded setup, and imagine having to set up 4 tables to get a DLQ per engine! Seemed like a bit much to me